kubernetes-config/locust-master-controller.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: "extensions/v1beta1"
+kind: "Deployment"
+metadata:
+  name: locust-master
```

Comment:
can we rename locust-master to locust-main or locust-lead or something along these lines? (more inclusive naming)

Reply:
sure, will do. potentially more problematic is the Web UI displays "number of slaves" so i'm not super sure how to deal with that

Reply:
done